packages/google_maps_flutter/google_maps_flutter_ios/pigeons/messages.dart

```diff
@@ -362,6 +382,8 @@ abstract class MapsInspectorApi {
   bool isTrafficEnabled();
   @ObjCSelector('getInfoForTileOverlayWithIdentifier:')
   PlatformTileLayer? getTileOverlayInfo(String tileOverlayId);
+  @ObjCSelector('getInfoForHeatmapWithIdentifier:')
```

Comment:
The one above is wrong; neither should have the `get`. And actually since we are using typed objects, we don't need "info" either, this should just be called `heatmapWithIdentifier:` (and the one above `tileLayerWithIdentifier:`, but you don't have to fix that in this PR).